recipes/pydstool/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "0.90.2" %}
+{% set sha256 = "d2603069927f4e6f69038f90a993f810c63585b6bf428473bcd5254d940a449f" %}
```

Comment:
Change this to `6e0cbceab8088ee8cbd0839202cdbaeebbe4ab2e6d1a0d0053c7de30268587c1`

Got it from running `sha256sum` on the downloaded archive.